Sources/Fullscreen/EmptyView/EmptyView.swift

```diff
@@ -5,6 +5,10 @@
 import CoreMotion
 import UIKit
 
+public protocol EmptyViewDelegate: class {
+    func performAction(from emptyView: EmptyView)
```

Comment:
I think this naming would be more consistent with the other delegates.

```swift
func emptyView(_ emptyView: EmptyView, didSelectActionButton button: Button)
```

Reply:
Ah, yes. Consistency is King 🤴🏻